index.html

```diff
@@ -0,0 +1,154 @@
+            </ul>
+          </div>
+          </div>  
+          <div class="nav-bars">
+            <ul class="mobile-menu-option">
+              <li id="close-menu"><i class="fa fa-times" aria-hidden="true">
+              </i></li>
+              <li><a href="index.html" class="navLink">Home</a></li>
+              <li><a href="about.html" class="navLink">About</a></li>
+              <li><a href="#" class="navLink">Program</a></li>
+              <li><a href="#" class="navLink">Join</a></li>
+              <li><a href="#" class="navLink">Sponser</a></li>
+              <li><a href="#" class="navLink">News</a></li>
+              <li><button>CC Campiaign</button></li>
+            </ul>
+          </div>
+      </header>
+      <div>
+        <div class="intro">
+          <div class="intro-content">
+            <h3>Welcome to Faya Beauty Salon</h3>
+            <h1>COME HERE TO LOOK <br> GLOW and <br>SHINE</h1>
+            <p class="para1">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam praesentium odio porro deserunt, nulla officia molestias repellendus.</p>
+            <h2>Open Monday~Saturady</h2>  
+            <p class="address">@Kilinto conduminium around commersial bank of ethiopia third floor</p>
+          </div>
+        </div>
+        <div class="main-program">
+          <h4>Main Program</h4>
+          <hr class="horizontal-line">
+          <div class="card-list">
+            <div class="card">
+              <img src="image/lecture.png" alt="">
+              <h5>Screen</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/exhibition.png" alt="">
+              <h5>Bridal</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/forum.png" alt="">
+              <h5>Fashion</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/workshop.png" alt="">
+              <h5>On Screen</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+            <div class="card">
+              <img src="image/ccignite.png" alt="">
+              <h5>On Screen</h5>
+              <p>Services are avaliable for you at any time and first you should have to book then we will send you the avliable slot</p>
+            </div>
+          </div>
+          <a href="" class="see-whole-program">SEE THE WHOLE PROGRM</a>
+          <Button type="button">Book Now for Your Program</Button>
+        </div>
```

Comment:
- [ ] Nice implementation of the main section of your app but you still need to work on your design to meet that of the required design.
![main](https://user-images.githubusercontent.com/55596970/170737610-fe1b4076-bd2a-4ef4-8507-af43936efc0d.png)
### **Your design**
![yourMain](https://user-images.githubusercontent.com/55596970/170738210-f2fbe82d-1167-4ddf-938d-f41011820923.png)